### ISSUE TITLE
Bug: Sisu version 5.0 breaks the extension

### ISSUE THREAD

**Issue by lamtonylam** [ApsiV11/sisu-grade-distribution#4]
New Sisu version breaks the extension as it no longer displays the grade distribution

**Comment by aapolaakkio** [comment]
Yep this is the case. Seems like Funidata finally stopped fetching the grade distribution to the SISU frontend. Thus the extension won't work anymore.

I guess we could try fetching the data ourselves but Funidata probably has removed the endpoint completely.

**Comment by aapolaakkio** [comment]
Actually I was wrong. The url path had just changed. Fixed now and will release new version

**Comment by lamtonylam** [comment]
Great! Thank you!
